### ISSUE TITLE
Lost Content when click "OK" in the Options Dialog

### ISSUE THREAD

**Issue by JanBrandenburger-LSF** [iceteagroup/Wisej-Ext-Kendo#2]
Hi ITG-Team,
we have a very strange issue.
We add a Kendo Wrapper Control and open the Options Dialog.
We insert JavaScript Content and click ok. When reopen, some lines are removed.
We checked it up with the telerik documentation and used the sample (https://docs.telerik.com/kendo-ui/api/javascript/dataviz/ui/map/configuration/layers.tooltip.template).

Example when insert this in a map control:
{
    "center": [45, 45],
    "zoom": 4,
    "layers": [{
        "type": "tile",
        "urlTemplate": "http://#= subdomain #.tile.openstreetmap.org/#= zoom #/#= x #/#= y #.png",
        "subdomains": ["a", "b", "c"],
        "attribution": "&copy; <a href='http://osm.org/copyright'>OpenStreetMap contributors</a>"
    }, 
    {
        "type": "marker",
        "tooltip": {
            "template": "#= kendo.format('{0}: {1:c0}',data.marker.dataItem.Land,data.marker.dataItem.Betrag) #"
    },
        "locationField": "Location",
        "titleField": "Betrag",
        "style": {
            "fill": {
                "color": "#fff",
                "opacity": 0.4
            }
        }
    }]
}

When reopen the part  "tooltip": {
            "template": "#= kendo.format('{0}: {1:c0}',data.marker.dataItem.Land,data.marker.dataItem.Betrag) #" was removed.

There was no error message, nothing.

In other Kendo Wrapper (chart) we have this effect when using "Label"-Option.

We have also observed that it happens that after a passage you put an enter or no enter - sometimes it is accepted and sometimes content is removed.

Do you make an attempt to perform a live validation that this effect occurs?

Have you any idea to avoid this behaviour.

At the first time, the work of 20 minutes was lost with one "ok"-click.

Thank you!

Jan

**Comment by lucapivato** [comment]
Thank you. Yes it’s an issue we had for a while but didn’t find a fix yet.

What happens is that in some cases, especially with complex maps and when the change is only in a child map, the dynamic object seriallizer doesn’t detect that the value has changed and it doesn’t save it.

A potential workaround is to also change a field in the main map. Even a non existing field.

I hope we’ll have the fix soon.

**Comment by JanBrandenburger-LSF** [comment]
Ah okay. Thank you for your quick answer.
I´ll try to use your workaround.

**Comment by JanBrandenburger-LSF** [comment]
Hi Luca, @lucapivato 
is there any progress  solution for the bug or is it maybe fixed already?

Best,
Jan

**Comment by lucapivato** [comment]
Nope, it's very hard to reproduce. It works for me 99.9% of the times.
